### PR TITLE
fix terms in CAPI for OpenStack RNs

### PR DESCRIPTION
**Pull request by jeana-redhat**
Version(s):
4.15

Issue:
Follows up on #70182 and #70813 for [OSDOCS-9302](https://issues.redhat.com//browse/OSDOCS-9302)

Link to docs preview:
- [OpenStack integration into the Cluster CAPI Operator (Tech Preview)](https://75024--ocpdocs-pr.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-15-release-notes.html#ocp-4-15-capi-openstack-tech-preview-no-upgrade)
- [Red Hat OpenStack Platform (RHOSP) Technology Preview features](https://75024--ocpdocs-pr.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-15-release-notes.html#red-hat-openstack-platform-rhosp-technology-preview-features)

QE review:
N/A

Additional information:

### CODE REVIEW

File: release_notes/ocp-4-15-release-notes.adoc [hunk 1]
```diff
@@ -75,12 +75,14 @@ For more information, see the link:https://access.redhat.com/documentation/en-us
...
+==== OpenStack integration into the Cluster CAPI Operator (Tech Preview)
 
-If you enable the `TechPreviewNoUpgrade` feature flag, the Cluster API (CAPI) Operator deploys the Cluster API Provider for OpenStack (CAPO) and manages its lifecycle. The CAPI Operator automatically creates `Cluster` and `OpenStackCluster` resources for the current {product-title} cluster.
+If you enable the `TechPreviewNoUpgrade` feature flag, the Cluster CAPI Operator deploys the Cluster API Provider OpenStack and manages its lifecycle.
+The Cluster CAPI Operator automatically creates `Cluster` and `OpenStackCluster` resources for the current {product-title} cluster.
 
-It is now possible to configure the CAPI `Machine` and CAPO `OpenStackMachine` resources in similar fashion to how Machine API (MAPI) resources are configured. It is important to note that the CAPI resources are equivalent to MAPI resources but not identical.
+It is now possible to configure the Cluster API `Machine` and `OpenStackMachine` resources similarly to how Machine API resources are configured.
+It is important to note that the Cluster API resources are equivalent to Machine API resources but not identical.
```
Comment (nrb) [inline release_notes/ocp-4-15-release-notes.adoc:85]: "equivalent but not identical" is a little confusing.

Maybe "functionally equivalent but not structurally identical?" They share the same purpose, but not the same fields.

Comment (jeana-redhat) [reply release_notes/ocp-4-15-release-notes.adoc]: revised, thanks!

File: release_notes/ocp-4-15-release-notes.adoc [hunk 2]
```diff
@@ -75,12 +75,14 @@ For more information, see the link:https://access.redhat.com/documentation/en-us
 
 You can now encrypt Azure storage accounts during installation by providing the installation program with a customer managed encryption key. See xref:../installing/installing_azure/installation-config-parameters-azure.adoc#installation-configuration-parameters-additional-azure_installation-config-parameters-azure[Installation configuration parameters] for descriptions of the parameters required to encrypt Azure storage accounts.
 
-[id="ocp-4-15-CAPO-tech-preview-no-upgrade"]
-==== CAPO integration into the cluster CAPI Operator (Tech Preview)
+[id="ocp-4-15-capi-openstack-tech-preview-no-upgrade"]
+==== OpenStack integration into the Cluster CAPI Operator (Tech Preview)
```
Comment (dfitzmau) [inline release_notes/ocp-4-15-release-notes.adoc:79]: The common-attributes.adoc file list:

:rh-openstack-first: OpenStack
:rh-openstack: OpenStack

Use these in place of hardcoded "OpenStack" references?

File: release_notes/ocp-4-15-release-notes.adoc [hunk 3]
```diff
@@ -75,12 +75,14 @@ For more information, see the link:https://access.redhat.com/documentation/en-us
...
 
-If you enable the `TechPreviewNoUpgrade` feature flag, the Cluster API (CAPI) Operator deploys the Cluster API Provider for OpenStack (CAPO) and manages its lifecycle. The CAPI Operator automatically creates `Cluster` and `OpenStackCluster` resources for the current {product-title} cluster.
+If you enable the `TechPreviewNoUpgrade` feature flag, the Cluster CAPI Operator deploys the Cluster API Provider OpenStack and manages its lifecycle.
```
Comment (dfitzmau) [inline release_notes/ocp-4-15-release-notes.adoc:81]: "Cluster API Provider OpenStack" feels like something is missing, how about add "implementation" at the end as "OpenStack" is largely a platform and "Cluster API Provider OpenStack" sounds like an API or Operator name.

File: release_notes/ocp-4-15-release-notes.adoc [hunk 4]
```diff
@@ -75,12 +75,14 @@ For more information, see the link:https://access.redhat.com/documentation/en-us
...
+The Cluster CAPI Operator automatically creates `Cluster` and `OpenStackCluster` resources for the current {product-title} cluster.
 
-It is now possible to configure the CAPI `Machine` and CAPO `OpenStackMachine` resources in similar fashion to how Machine API (MAPI) resources are configured. It is important to note that the CAPI resources are equivalent to MAPI resources but not identical.
+It is now possible to configure the Cluster API `Machine` and `OpenStackMachine` resources similarly to how Machine API resources are configured.
```
Comment (dfitzmau) [inline release_notes/ocp-4-15-release-notes.adoc:84]: "It is" can be vague referrents. How about:

> You can now configure...

File: release_notes/ocp-4-15-release-notes.adoc [hunk 5]
```diff
@@ -75,12 +75,14 @@ For more information, see the link:https://access.redhat.com/documentation/en-us
...
+It is important to note that while Cluster API resources are functionally equivalent to Machine API resources, structurally they are not identical.
```
Comment (dfitzmau) [inline release_notes/ocp-4-15-release-notes.adoc:85]:
```suggestion
Note that while Cluster API resources are functionally equivalent to Machine API resources, structurally they are not identical.
```

**Comment by dfitzmau** [comment]
Hi @jeana-redhat . Added some minor feedback inline. Nice work! 

/remove-label peer-review-in-progress

/label peer-review-done

File: release_notes/ocp-4-15-release-notes.adoc (hunk 2 above)
Comment (jeana-redhat) [reply release_notes/ocp-4-15-release-notes.adoc:79]: I can use them for some but not all instances of "OpenStack", so I do that for the ones that make sense :+1: (I have a [card](https://issues.redhat.com/browse/OSDOCS-10442) to make attributes for the full "Cluster API Provider OpenStack" and all its friends, so I will leave those ones alone)

File: release_notes/ocp-4-15-release-notes.adoc (hunk 3 above)
Comment (jeana-redhat) [reply release_notes/ocp-4-15-release-notes.adoc:81]: > feels like something is missing

I agree, but also that's what it's called :shrug:

**Pull request merged**